examples/with-antd-less/readme.md

```diff
@@ -0,0 +1,68 @@
+### Less loader for next.js and Antd integration.
+
+### 1. Example installation
+
+for this example, you can use commands as follows:
+
+```shell
+yarn install
+yarn dev
+```
+
+### 2. Project installation(for production project)
+
+#### 1. first step, install loader package
+
+```shell
+
+// using yarn
+yarn add next-antd-aza-less --save
+
+// or use npm
+npm install next-antd-aza-less --save
+
+```
+
+#### 2. second step, add .babelrc in your project
+
+```json
+{
+  "presets": ["next/babel"],
+  "plugins": [
+    [
+      "import",
+      {
+        "libraryName": "antd",
+        "libraryDirectory": "lib",
+        "style": true
+      }
+    ]
+  ]
+}
+```
+
+#### 3. change your .next.config.js
+
+Example:
+
+```javascript
+const antdLessLoader = require("next-antd-aza-less");
+const modifyVars = require("./your/custom/vars");
+
+if (typeof require !== "undefined") {
+  require.extensions[".less"] = file => {};
+}
```

Comment:
Could you find an alternative config for this? During export we only require `next.config.js` once and it is not loaded inside the workers causing this to throw a `SyntaxError` when a less file is required. 

The other examples also use this and issues have been opened due to it. See: #6867 #6384 #6181

Reply:
I'm still working on this ,and actually there are several problems needed to be fixed.
Just like:
1. Using require.extension will be warned by yarn lint;
2. Will there be any trouble after deleting the less extension on require property.